decidim-core/app/views/layouts/decidim/_redesigned_application.html.erb

```diff
@@ -7,11 +7,13 @@
   </head>
 
   <body class="text-black text-md form-defaults">
+    <!--noindex--><!--googleoff: all-->
```

Comment:
What does it means?

Reply:
No idea, I just copy-pasted what was in develop to avoid conflicts :man_shrugging:  I presume... SEO stuff :sweat_smile: